src/applications/static-pages/facilities/OtherFacilityListWidget.jsx

```diff
@@ -59,17 +59,19 @@ export default class OtherFacilityListWidget extends React.Component {
             </h3>
             <FacilityAddress facility={facility} />
             <div className="vads-u-margin-bottom--0">
-              <div className="main-phone vads-u-margin-bottom--1">
-                <strong>Main phone: </strong>
-                <a
-                  href={`tel:${facility.attributes.phone.main.replace(
-                    /[ ]?x/,
-                    '',
-                  )}`}
-                >
-                  {facility.attributes.phone.main.replace(/[ ]?x/, '')}
-                </a>
-              </div>
+              {facility.attributes.phone.main && (
+                <div className="main-phone vads-u-margin-bottom--1">
+                  <strong>Main phone: </strong>
+                  <a
+                    href={`tel:${facility.attributes.phone.main.replace(
```

Comment:
Is there a way to put a comment in here saying what this regex is attempting to do? Would make it easier for the next person.

Reply:
comment added